dataactvalidator/config/sqlrules/c23_award_financial_3.sql

```diff
@@ -1,3 +1,9 @@
+WITH award_financial_c23_3_{0} AS
```

Comment:
Just as a reference to the previous comment above, we should be consistent here. If we're using `WITH award_financial_c23_3_{0}` with the `{0}`, that should be used in the other SQL rules as well. If not, we should change this to be `WITH award_financial_c23_3`